ddtrace/settings/_database_monitoring.py

```diff
@@ -46,7 +48,7 @@ def _get_dbm_comment(db_span):
     }
 
     if dbm_config.injection_mode == "full":
-        # TODO: add _dd.dbm_trace_injected tag to db_span
+        db_span.set_tag_str(DBM_TRACE_INJECTED_TAG, "true")
```

Comment:
this seems like a weird side effect of this method. especially given that it is written where it seems like calling this only returns generated data.

is there another common place we could consider setting this tag?

Reply:
I left the tag here to keep things concise. I don't see an alternative a side from moving this logic to it's own function and calling it in dbapi/aiomysql/aiopg/asyncpg. 

I have no strong feelings

Reply:
🤔 yeah, I am not sure the best action to take here. 

one other way to look at it is... this isn't _actually_ injecting the context into the sql statement, it is just generating the sql comment that would be injected. so are we sure it is going to get used and the injection will occur?

Reply:
I'll add a docstring to note the side effect